[ready] NSV Jeppison

Vasily2013 opened the pull request:
## About The Pull Request

Incoming message from Nanotrasen Fleet Command. A new ship classification has been prototyped and is ready for testing. This ship focuses heavily on ballistic, non-guided munitions and a minimal fighter compliment. 

## Why It's Good For The Game

Vago needs a partner

## Changelog
:cl:
add: A whole new map!
Tweak: Adds some hull plating to FOB! 
/:cl:

Firefox13 commented:
Add an anterchamber with some plasteel(to fix), radiation closet and some fluff stuff
![изображение](https://user-images.githubusercontent.com/57810301/81127083-c642bf80-8f45-11ea-85a9-7108e16aa753.png)
Eh?
![изображение](https://user-images.githubusercontent.com/57810301/81127966-8d581a00-8f48-11ea-9100-6a741d6bf767.png)
.
Where the unfiltered gas(not plasma, O2, CO2, N2, N2O) goes?
.
Either remove the water cooler or add some plastic cups on the table near it
![изображение](https://user-images.githubusercontent.com/57810301/81127135-ef635000-8f45-11ea-97e6-1c9437be6259.png)
INJECTOR, NOT VENT
![изображение](https://user-images.githubusercontent.com/57810301/81127177-10c43c00-8f46-11ea-94ac-fc5f88b7c3c2.png)
Maybe put those crates into warehouse?
![изображение](https://user-images.githubusercontent.com/57810301/81127222-34878200-8f46-11ea-8770-4150b64de4fc.png)
This place lacks either blood or synthetic limbs crate
![изображение](https://user-images.githubusercontent.com/57810301/81127240-4b2dd900-8f46-11ea-99ea-7b5423a1f215.png)
The fuck is this floor? Also, put a cryo beaker near cryo chamber since not everyone got access to morgue
![изображение](https://user-images.githubusercontent.com/57810301/81127260-5c76e580-8f46-11ea-987e-f0d564ec0212.png)
.
I think chemistry lacks pill bottles
.
Sparks?
![изображение](https://user-images.githubusercontent.com/57810301/81128089-d60fd300-8f48-11ea-9e8e-f3d53320ebad.png)
Probably too much loot for 1 room? Gibs is a bit too much for maintenance either
![изображение](https://user-images.githubusercontent.com/57810301/81127333-8fb97480-8f46-11ea-8fda-fd199042b380.png)
I hope that's pink glitter, not plasma
![изображение](https://user-images.githubusercontent.com/57810301/81127346-99db7300-8f46-11ea-9f6f-41b3251ac3dd.png)
Can't reach the tank storage unit
![изображение](https://user-images.githubusercontent.com/57810301/81127412-c8f1e480-8f46-11ea-9987-3a7d7884cb01.png)
I cry as an Atmosian, getting jealous with that amount of canisters scientists have
![изображение](https://user-images.githubusercontent.com/57810301/81127439-df983b80-8f46-11ea-8844-a80098c89f7c.png)
Add showers and xenobio lockers to xenobio passage
![изображение](https://user-images.githubusercontent.com/57810301/81127463-f8085600-8f46-11ea-8192-3f1b4095052e.png)
Nice light
![изображение](https://user-images.githubusercontent.com/57810301/81127474-035b8180-8f47-11ea-8549-938e3c6edf6b.png)
Actually this entire corridor of space looks obsolete. Mind replacing it with maint tunnel?
![изображение](https://user-images.githubusercontent.com/57810301/81127504-1706e800-8f47-11ea-96d9-1e5144cfd8ea.png)
***Three*** techfabs
![изображение](https://user-images.githubusercontent.com/57810301/81127515-1f5f2300-8f47-11ea-9e7b-cdca7b02e3b7.png)
Not funny, didn't laugh
![изображение](https://user-images.githubusercontent.com/57810301/81127541-2f770280-8f47-11ea-9e30-9d503cf0d25c.png)
Bartenders, chemists, virologists will kill you for this ORM placement. What are roller beds for?
![изображение](https://user-images.githubusercontent.com/57810301/81127562-43baff80-8f47-11ea-999d-f6f71c34ce91.png)
Make materials more organized here, please
![изображение](https://user-images.githubusercontent.com/57810301/81127585-5cc3b080-8f47-11ea-9824-aa356af555fc.png)
...why?
![изображение](https://user-images.githubusercontent.com/57810301/81127614-7238da80-8f47-11ea-92df-b347a1d2697e.png)
D E C A L S
![изображение](https://user-images.githubusercontent.com/57810301/81127629-7ebd3300-8f47-11ea-9f36-ec6f4ceb1a85.png)
Imagine falling down right into medbay
![изображение](https://user-images.githubusercontent.com/57810301/81127653-92689980-8f47-11ea-8c92-f4015c308e55.png)
.
Add railings to the stairs
.
I think this place lacks cables
![изображение](https://user-images.githubusercontent.com/57810301/81127694-b5934900-8f47-11ea-9329-16dd6892b3cc.png)
Add tiles
![изображение](https://user-images.githubusercontent.com/57810301/81127723-c774ec00-8f47-11ea-8cbe-3916ede74a48.png)
Replace glass table with reinforced table
![изображение](https://user-images.githubusercontent.com/57810301/81127732-d065bd80-8f47-11ea-84ee-27bd51ffac58.png)
Prisoners ***will*** break into warden's office. The fuck is mass driver for? So unconfirmed changelings can escape?
![изображение](https://user-images.githubusercontent.com/57810301/81127750-e3788d80-8f47-11ea-83c5-fbec043dea9f.png)
.
4+4 lockers. 8 sec officers, you seriously?
.
ez sec loot
![изображение](https://user-images.githubusercontent.com/57810301/81127776-04d97980-8f48-11ea-80c1-4bf4f6c7fbb3.png)
SHUTTERS
![изображение](https://user-images.githubusercontent.com/57810301/81127789-0efb7800-8f48-11ea-98a1-fc74e120a700.png)
You lack a glass here
![изображение](https://user-images.githubusercontent.com/57810301/81127833-276b9280-8f48-11ea-8878-f45878647950.png)
Telecomms **will** overheat
![изображение](https://user-images.githubusercontent.com/57810301/81127848-35211800-8f48-11ea-99d0-a8b04bcf62e1.png)
Wiring?
![изображение](https://user-images.githubusercontent.com/57810301/81127869-4702bb00-8f48-11ea-957f-1867e350ab16.png)
Why there are handcuffs?
![изображение](https://user-images.githubusercontent.com/57810301/81127882-4ff38c80-8f48-11ea-93ce-125c15aea828.png)
***T I L E S***
![изображение](https://user-images.githubusercontent.com/57810301/81128153-15d6ba80-8f49-11ea-8a43-33390ae329aa.png)
overpriced
![изображение](https://user-images.githubusercontent.com/57810301/81127915-6994d400-8f48-11ea-8042-1132a91ad32e.png)
Merge this into one long tunnel or fill both rooms with some shit
![изображение](https://user-images.githubusercontent.com/57810301/81128196-40287800-8f49-11ea-84b8-20feb28f8991.png)
>Enforcing MRP
>Putting a fucking nuclear warhead in MaA office
![изображение](https://user-images.githubusercontent.com/57810301/81128182-31da5c00-8f49-11ea-96f0-9f2a61cb0c5c.png)
.
All hallways look very empty. Add decals, signs, vendors
.
What are torpedoes for? Add torpedo tubes
...
I do actually like this map

Vasily2013 commented:
Alright, adressing your critiques. 

Good ideas I like
-A larger airlock to grav gen could be fun, would also give a place for a SMES
-change toxins room to injector (This was a toxins coppied from another map)
-I'll move some crates into the warehouse.
-Lockers and showers to xenobio room
-remove light in toxins and direct it against the wall.
-find better place for ORM placement, also design anti-theft system
-decals
-signs
-add pets
-fully secure off the AI law boards.
-Add tiles to chapel (Look at commit history. I literally did this last)
-add a locker or two above MAA's office
-Torps were there before DI2, Will change to missiles later to help the fighter compliment.

Rebuttles to points I don't.
-Your rendering program isn't rendering wires
-not sure what "eh" means. That room is specifically for the plasma constrictors.
-Unfiltered gas goes outside. This can be redirected elsewhere as a fun in-round project.
-I'm about 90 percent sure that water coolers start with a bunch of paper cups.
-That crate is blood, look in the medical supply/storage for limbs
-Those sparks are a gib spawner
-Room copied off Box
-It's neither!
-You can move the firefoam or water tank depending on what you need.
-atmos can always ask to borrow some. In general though this is because I have no clue as to what toxins actually needs, so "enough" is my standard.
-that "corridor" is specifically to make it so cargo doesn't share a wall with toxins.
-Actually that's techfab, protolathe, protolathe
-that is actually glitter, and I think it's entertaining. I could change it to plasma.
-Roller beds are for dead miners.
-You think a NT restocking crew gives a damn about organization?
-Prevents high security armory from having shared wall with a public hallway.
-You actually fall into the Maint or the MAA office.
-Stairs have railings
-Janitors can add tiles
-If they try to break into the warden's office the warden shouldn't be too far away... as well the warden should secure his equipment better if they get it.
-I had extra space in sec. Got suggestions for one of those storage rooms?
-Lockers are anchored down, and secure your lockers. Those will also probably be the first pillaged by security officers.
-HOS has shutters. Button is over disposals.
-Tcomms overheating isn't a thing in this codebase.
-Handcuffs are obviously for throwing degenerates off of the bridge.
-"Overpriced?" I say Command office.
-The nuke has a name, its name is Antonio.

ta3370 commented:
If that ORM placement is a complaint, you're gonna HATE concordat xD. What do people have against _asking and communicating with others_ ? Inspiring communication should always be a goal, plus having wall-ORM *ALWAYS* leads to spergs using it to break into cargo, even further reducing communication.

Firefox13 commented:
> -not sure what "eh" means. That room is specifically for the plasma constrictors.

![изображение](https://user-images.githubusercontent.com/57810301/81132905-aa94e480-8f58-11ea-83c0-73a16e083060.png)
I actually meant this scrubber, but it looks like all scrubbers doubled. Stupid map diff render.
> -I'm about 90 percent sure that water coolers start with a bunch of paper cups.

Strangely, they do. Got really used to a box of paper cups near water cooler
![изображение](https://user-images.githubusercontent.com/57810301/81133267-c0ef7000-8f59-11ea-960d-6b59856cd618.png)
> -I had extra space in sec. Got suggestions for one of those storage rooms?

A small office area.
> -Lockers are anchored down, and secure your lockers. Those will also probably be the first pillaged by security officers.

There is no delay in unwrenching the closet. Don't really understand the idea of security closets behind a single door. 
> -If they try to break into the warden's office the warden shouldn't be too far away... as well the warden should secure his equipment better if they get it.

Isn't this map is meant for low pop? Big chance we don't get a warden.

> -Handcuffs are obviously for throwing degenerates off of the bridge.

Put a box of handcuffs, then.
.
I'm fine with rest of the answers.
.
No blood bags?
![изображение](https://user-images.githubusercontent.com/57810301/81133634-e335bd80-8f5a-11ea-8531-11eb389c20b6.png)

Vasily2013 commented:
-Theoretically the bridge shouldn't need more than one pair of handcuffs at a time, and they should recover equipment from security if they use one. Two should be enough.
-added some blood to brigbay
-added some perma shutters

Bokkiewokkie commented:
Why does the service sector have a total of three lathes all located within 30 tiles of eachother? The cook and the bartender can just walk to the service hall wich is literally at the end of the hallway.

Vasily2013 commented:
> Please talk to KMC about the way the overmap is handled now. I believe we do not use the overmap.dmm as a direct thing for each map anymore.

Map is on hold until starmap is merged specifically due to the fact of everything is changing.

Vasily2013 commented:
Please place all other issues and commentary here. 
https://github.com/BeeStation/NSV13/issues/608

francinum commented on _maps/map_files/Jeppison/job_changes.dm:
```diff
@@ -0,0 +1,13 @@
+	total_positions = 2
+	spawn_positions = 2
+
+/datum/job/deck-tech/new()
```
Improper capitalization
```suggestion
/datum/job/deck_tech/New()
```

francinum commented on _maps/map_files/Jeppison/job_changes.dm:
```diff
@@ -0,0 +1,13 @@
+/datum/job/pilot/New()
```
```suggestion
/datum/job/fighter_pilot/New()
```

Firefox13 commented:
> Keep your chain of command filled.

like people ever care for it

I think it requires a light to see your bomb
![изображение](https://user-images.githubusercontent.com/57810301/82812753-5cf70200-9e9c-11ea-9d0b-d467b809f6be.png)

I also found a telecrystal in cargo. It's either very cringe random spawn or you put it there. Really not a good idea!

wait wait is that random maint spawns? or circuits? please make it circuits!
![изображение](https://user-images.githubusercontent.com/57810301/82812848-8d3ea080-9e9c-11ea-961a-bd2c76b19d97.png)

"""airlocks""" lack scrubbers and vents
![изображение](https://user-images.githubusercontent.com/57810301/82812911-ad6e5f80-9e9c-11ea-8040-af173f8c6b6a.png)

...can those rooms get increased to 3x3?
![изображение](https://user-images.githubusercontent.com/57810301/82812939-c24af300-9e9c-11ea-8ea7-c8f45137e6a7.png)

Add some maints there or transform it into bar backroom
![изображение](https://user-images.githubusercontent.com/57810301/82813002-ddb5fe00-9e9c-11ea-999d-e6a9069417a2.png)

can we get stabilized ration packs from Enterprise?
![изображение](https://user-images.githubusercontent.com/57810301/82813042-f58d8200-9e9c-11ea-95b7-16f569688f57.png)

can you please organize it, i.e. it's own tile for every medkit type. also lacks oxygen medkits
![изображение](https://user-images.githubusercontent.com/57810301/82813103-11912380-9e9d-11ea-9c8b-0ce729687545.png)

add maint entry and those fucking things I forgot their names to windows
![изображение](https://user-images.githubusercontent.com/57810301/82813201-443b1c00-9e9d-11ea-8f9d-ca1eedf4c1ae.png)
these things, for "privacy", so you don't get distracted ~~while you amputate cap's head~~
![изображение](https://user-images.githubusercontent.com/57810301/82813262-5e74fa00-9e9d-11ea-8d1b-d5d9243f932b.png)

.
I think robotics lacks plasteel.
.

Vasily2013 commented:
- I'm not moving perma.
- [ ] Light added to the bomb testing area
- that TC would be from a random maint spawner. Not sure if it was real or not.
- Those are board spawners for tech storage.
- [x] Scrubbers and vents added
- No
- The reason their is no maint there is because Virology, a rather sensitive area, is directly below it.
- [x] O2 deprivation kits added, sorting is up to player discretion due to there being 5 types of medkits and only 4 cells.
- [x] Added curtains to surgery windows
- If you show me a decent way to put a maint airlock in there feel free to show me, because I don't see one that's pretty, and otherwise you have to run through very little space to get to a maint airlock.
- [x] Added a small amount of round start plasteel to robotics positions.
